rapids-cmake/cpm/versions.json

```diff
@@ -12,10 +12,10 @@
       "git_url" : "https://github.com/fmtlib/fmt.git",
       "git_tag" : "${version}"
     },
-    "GBench" : {
-      "version" : "1.5.3",
+    "benchmark" : {
```

Comment:
Can we move this up, to keep it alphabetical? `benchmark` before `cuco`.

Reply:
Done